pcsx2/PAD/Host/KeyStatus.cpp

```diff
@@ -107,6 +107,7 @@ void KeyStatus::Set(u32 pad, u32 index, float value)
 			1, // PAD_R2
 			9, // PAD_L3
 			10, // PAD_R3
+			16, // Analog
```

Comment:
Magic number here too, but we could probably just leave this one since the rest also aren't using the enum either... (I think it was originally a different order here, until I synced them up, the onepad code was quite... gross).

Reply:
*LilyPad code sweating in bad code*